Okta.AspNetCore.Mvc.IntegrationTest/OktaMiddlewareShould.cs

```diff
@@ -56,6 +56,19 @@ public async Task IncludeIdpInAuthorizeUrlAsync()
             }
         }
 
+        [Fact]
+        public async Task IncludeLoginHintInAuthorizeUrlAsync()
+        {
+            var loginWithIdpEndpoint = string.Format("{0}/Account/LoginWithLoginHint", BaseUrl);
+            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, loginWithIdpEndpoint);
```

Comment:
Looks like this is unnecessary line: the `request` variable is not used anywhere.

Reply:
Good catch! I'll fix it. 👍

Reply:
Fixed.